### PR TITLE
Add help command to makefile and .PHONY

### PR DESCRIPTION
**Pull request by pando85**

### CODE REVIEW

**Comment by galexrt** [comment]
@pando85 Could you change your PR to implement the `help` target as in the main repository, see https://github.com/rook/rook/blob/master/Makefile#L181-L186
For this repo the `HELPTEXT` part isn't needed.

**Comment by pando85** [comment]
Ok, feel free to close it.

**Comment by pando85** [comment]
> @pando85 Could you change your PR to implement the `help` target as in the main repository, see https://github.com/rook/rook/blob/master/Makefile#L181-L186
> For this repo the `HELPTEXT` part isn't needed.
Done

File: Makefile [hunk 1]
```diff
@@ -9,15 +18,15 @@ run: _data/projects.json
 		$(JEKYLL_DOCKER_IMAGE) -- \
 		jekyll serve --livereload --livereload-port 4001
 
-# Build (output is in _site)
+build:	## build output is in _site
```
Comment (galexrt) [inline Makefile:21]: One last thing I just noticed, please either use tabs or spaces for the comments after the target name.
E.g., here a tab is used, but for the help and run target spaces are used to separate the target name from the comment `##`.
After that I'll go ahead and merge it.

Comment (pando85) [reply Makefile:21]: Sorry, a problem of consistence. Done

**Comment by galexrt** [comment]
@pando85 No problem! Thank you for making the changes!

The DCO check is failing now, please squash your two commits into one and make sure the DCO sign off is applied correctly (see https://github.com/rook/rook/blob/master/CONTRIBUTING.md#certificate-of-origin).
(your first commit has the correct sign off on it, so squashing and removing the second commit's message should fix it)

**Comment by pando85** [comment]
Here we go!

**Pull request merged**